File: starters/rmsbolt.swift
import Foundation

// Swift rmsbolt starter file

// Local Variables:
// rmsbolt-disassemble: nil
// End:

func isRMS(_ a: Character) -> Int {
    switch (a) {
    case "R":
        return 1
    case "M":
        return 2
    case "S":
        return 3
    default:
        return 0
    }
}

func main() -> Int {
    let a: Character = "N"
    if isRMS(a) == 0 {
        print(a)
    }
    return 0
}

File: starters/rmsbolt.f90
! Fortran 90/95/2003/2008/2018 starter file
!
! Local Variables:
! rmsbolt-command: "gfortran -O0"
! rmsbolt-disassemble: nil
! End:

program main

    implicit none

    character :: a

    a = 'A'

    if( isrms( a ) ) then
        write( *, '(A)' ) a
    end if

contains

    function isrms( a )

        implicit none

        character :: a
        logical   :: isrms

        if( a .eq. 'R' ) then
            isrms = .true.
        else if( a .eq. 'M' ) then
            isrms = .true.
        else if ( a .eq. 'S' ) then
            isrms = .true.
        else
            isrms = .false.
        end if

    end function isrms

end program main
